shared/src/main/java/org/odk/collect/shared/PathUtils.kt

```diff
@@ -0,0 +1,22 @@
+package org.odk.collect.shared
+
+import java.io.File
+
+object PathUtils {
+
+    @JvmStatic
+    fun getRelativeFilePath(dirPath: String, filePath: String?): String? {
+        if (filePath == null || filePath.isEmpty()) {
```

Comment:
Looking at it now, I'm actually wondering if we should just remove the check. I think it would a coding error for us to pass `null` into this, and I can't see anywhere where we'd genuinely want to return `null` from either of these (that's also the case for the `StoragePathProvider` functions that make heavy use of these helpers). I'd imagine if the logic in this method doesn't explode, returning `null` will (like passing it to a `File` constructor). 

What do you think? Should I go ahead and make them return `String` rather than `String?`.

Reply:
I've made the change but can easily revert the last two commits if you don't think it makes sense.